Yoast/wordpress-seo
# Introduce meta description presenter

## Summary

Introduces the first presenter for indexables. Intended primarily to serve as an example for future development.

This PR can be summarized in the following changelog entry:

* No user-facing changes.

## Relevant technical choices:

* Only simple pages are supported for now, in all other cases it falls back to the original behaviour.

## Test instructions
This PR can be tested by following these steps:

* Enable the `YOAST_SEO_EXPERIMENTAL_PHP56` and `YOAST_SEO_INDEXABLES` feature flags.
* The meta description on the front-end should remain identical.
* For every post you visit on the front-end a new entry in the indexables table should be created.

## Documentation

## Quality assurance
